Magefile.go

```diff
@@ -285,3 +298,9 @@ func (Run) SourceDiffLocal(ctx context.Context, archive string, source string) e
 	return sh.RunV(command[0], command[1:]...)
 
 }
+
+func (Run) IntegrationTests() error {
```

Comment:
I would move this target under the `Test` target, and also add a docstring so it's displayed when running `mage -l`,  as well as moving this up next to the other `(Test)` targets:

```suggestion
// Integration runs the integration tests for the plugin validator
func (Test) Integration() error {
```


This way it can be invoked via

```bash
mage test:integration
```

Which I think is more suited. WDYT?